.github/workflows/pull_request_check.yaml

```diff
@@ -0,0 +1,28 @@
+name: Pull Request Check
+on:
+  pull_request:
+    types: [opened, edited, reopened, synchronize]
+jobs:
+  pull_request_check:
+    name: pull request check
+    runs-on: ubuntu-latest
+    steps:
+      - uses: nearform/github-action-check-linked-issues@v1
```

Comment:
From the CI run:
```
Warning: The 'github-action-check-linked-issues' action, no longer exists under the 'nearform' organisation.
Please update it to 'nearform-actions', you can do this
by updating your Github Workflow file from:

    uses: 'nearform/github-action-check-linked-issues'

to:

    uses: 'nearform-actions/github-action-check-linked-issues'
```

Reply:
Yah i saw that but that link doesn't work.

Reply:
Hmm. Ok then.